_posts/platform/app/2000-01-01-log-drain.md

```diff
@@ -118,6 +118,21 @@ Where
 * `host`: host of the syslog server
 * `port`: port number of the syslog server
 
+### Add a Drain to an Addon
+
+In order to add a log drain to an addon, you need to get its UUID. List your
+application addons to get its UUID:
+```bash
+scalingo --app my-app addons
+```
+
+Then, use the option `--addon <uuid>` with the [add command](#add-a-log-drain):
+```bash
+scalingo --app my-app log-drains-add --addon ad-a86f546c-6b08-439b-a5fc-6f16ddc9083e \
+  --type syslog --host custom.logstash.com --port 12345
+```
+
+Currently, only database addons are able to forward logs to a drain.
```

Comment:
This is not completely true. All addons are able to forward logs to a drain. It's just that non-database addons do not generate logs, at all.